gcsfs/utils.py

```diff
@@ -119,4 +132,6 @@ def is_retriable(exception):
     errs += [str(e) for e in errs]
     if isinstance(exception, HtmlError):
         return exception.code in errs
+    if isinstance(exception, RateLimitException):
```

Comment:
Can you please include a link that says this type of exception is retriable? If I were google, I could imagine not recommending to retry calls that are already over some rate quota.

Reply:
Done. I will retry as client (papermill). Do you suggest the client owns retry logic as this PR? Or gcsfs should do the retries with backoff? (I see some logic in core GCSFileSystem _call method.

Reply:
It is correct that gcsfs should own the retries if we are convinced that the exception should indeed be retried in normal usage.

Reply:
I agree with your initial statement, for 429 Exception, GCS system allows only writing/updating object once per second. In papermill we were are sending more updates than that..., let me know if this PR is good for now or any other changes are recommended. Also how often the gcsfs package gets updated? Thanks

Reply:
I think you are right, and this work well from my point of view.